my_list.rb

```diff
@@ -0,0 +1,22 @@
+require_relative './my_Enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list
+  end
+
+  def each(&number)
+    @list.each(&number)
+  end
```

Comment:
- [ ] Inside `each` method you have to implement your logic to behave like each. kindly implement this.

Like this:

```
def each
    return to_enum(:each) unless block_given?

    counter = 0
    while counter < @list.length
      yield(@list[counter])
      counter += 1
    end
    @list
end
```